include/boost/static_string/static_string.hpp

```diff
@@ -729,6 +729,57 @@ defined(BOOST_STATIC_STRING_NO_PTR_COMP_FUNCTIONS)
     std::less<const T*>()(ptr, src_last);
 #endif
 }
+
+// This nested type workaround is for gcc 5,
+// which prohibits throw expressions in constexpr
+// functions, but for some reason permits them in
+// constructors.
+template<typename Exception>
+struct exception
```

Comment:
I would `#ifdef` out anything in detail namespaces (including the namespace declaration) when building the docs. There's no reason for doxygen to see those symbols, and sometimes doxygen has a bug which causes the doc build to fail.

Reply:
This is already hidden